templates/hooks/parametergroup/sdk_update_pre_build_request.go.tpl

```diff
@@ -1,7 +1,14 @@
-if delta.DifferentAt("Spec.ParameterNameValues") {
-    ko, err := rm.resetParameterGroup(ctx, desired, latest)
-
-    if ko != nil || err != nil {
-        return ko, err
+    if delta.DifferentAt("Spec.Tags") {
+        err = rm.updateTags(ctx, desired, latest)
+        if err != nil {
+            return nil, err
+        }
     }
-}
\ No newline at end of file
+
+    if delta.DifferentAt("Spec.ParameterNameValues") {
+        ko, err := rm.resetParameterGroup(ctx, desired, latest)
+
+        if ko != nil || err != nil {
+            return ko, err
+        }
+    }
```

Comment:
Thinking out loud:
I know this existed before. But it's a weird if/return statement for Go.
Suggestion for the future please rewrite `rm.setParameters` and make this il/else clearer :)